### ISSUE TITLE
Option to hide album name & year

### ISSUE THREAD

**Issue by prism2001** [phin05/discord-rich-presence-plex#83]
it would be nice if there was a config option to remove the album name and the year from the second line
![image](https://github.com/phin05/discord-rich-presence-plex/assets/11824514/bc472790-62f1-4b56-9ae3-d78066db190f)
so that only the artist name appears, similar to the built-in Spotify rich presence. makes for a cleaner look imo

**Comment by phin05** [comment]
I plan to add a templating system that allows customising all that.

In the meantime, to implement your preference, you can replace this line:

https://github.com/phin05/discord-rich-presence-plex/blob/5461ad20ed0172599e3a9d41cc0507811b3abe15/core/plex.py#L253

with

```python
stateStrings.append(item.originalTitle or item.grandparentTitle)
```

and (optionally) delete these lines:

https://github.com/phin05/discord-rich-presence-plex/blob/5461ad20ed0172599e3a9d41cc0507811b3abe15/core/plex.py#L249-L252

**Comment by phin05** [comment]
Implemented in [v2.8.0](https://github.com/phin05/discord-rich-presence-plex/releases/tag/v2.8.0).

Set `display.album` to `false` in your config.
